content/groups/brain.md

```diff
@@ -3,7 +3,7 @@ slug: brain
 name: Analytics & Intelligence Network
 next-meeting: "2019-06-27T17:30:00.000Z"
 icon: ../assets/brain.png
-group_url: https://batonrougeanalytics.pass.org/
+group_url: https://www.gobrain.org
```

Comment:
The certificate for https doesn't appear to be valid, or at least not signed by a recognized CA so recommend switching this to just use `http` for now.

```suggestion
group_url: http://www.gobrain.org
```